File: 11_intro_json_python_peer_review/program_files/part1.py
from splashkit import *

def main():

    game_data = json_from_file("game_data.json")
    game_title = json_read_string(game_data, "gameTitle")

    if not game_title:
        print("Error retrieving gameTitle.")
    else:
        print("Game Title:", game_title)
    
    free_json(game_data)


if __name__ == "__main__":
    main()

File: peer-reviews/11_intro_json_python_peer_review/program_files/part1.cpp
#include "splashkit.h"
int main()
{
    json game_data;
    game_data = json_from_file("game_data.json");
    string game_title = json_read_string(game_data, "gameTitle");

    write("Game Title: ");
    write_line(game_title);

    free_json(game_data);

    return 0;
}